### DISCUSSION TITLE
Add a separate option to Minimize the search popup only for input or textarea

### DISCUSSION THREAD

**Discussion by zhaocai** [hoothin/SearchJumper#52] [Ideas]
有时在输入框选择一段文字并不是为了搜索，继续输入的时候搜索酱Popup会遮盖文字预测，如下图：
![image](https://user-images.githubusercontent.com/341473/228726676-c9009697-2b35-4ad2-ba87-49082450aeb0.png)

能不能单独为输入框的Popup添加最小化弹出的选项。谢谢！

**Comment by hoothin** [comment]
设置里不是有选项吗？【监听输入框】这个，取消勾选就行了。取消之后你依旧可以通过【绑定工具栏唤出功能键】来在输入框内唤出

**Comment by zhaocai** [comment]
## 需求

1. 在输入框弹出如下图：
![image](https://user-images.githubusercontent.com/341473/228727791-dee57841-06ca-49d8-94ae-9136e01614a0.png)

2. 在其他地方划词弹出如下图：
![image](https://user-images.githubusercontent.com/341473/228728203-6295ae21-3ddb-4983-976c-d189703f1f02.png)

**Comment by hoothin** [comment]
相比我目前所用的【监听输入框禁用】+【绑定工具栏唤出功能键Ctrl】。这么改有什么效率上的提升吗？

**Reply by zhaocai** [reply]
单手鼠标就可以操作，左手可以休息^_^

**Comment by hoothin** [comment]
好的，谢谢提议，我加个选项

**Reply by zhaocai** [reply]
顺便能不能给这个添加一个缩放选项 ![image](https://user-images.githubusercontent.com/341473/228734420-db6c07f6-5b91-4191-a87c-a033d1cb1391.png)
不好截图就用网站上面这个示例，我这里显示有点小。

**Comment by zhaocai** [comment]
问题+1...

## 一个图标聚合多开时单击无响应

如下，一个图标多开搜索几个网站，但是在划词搜索弹出时单击无响应，试了一会儿才发现必须要按着`Ctrl`才能响应并打开多个Tab进行搜索。还是和前面说的一样单手操作的问题；单击无响应是为什么呢？是这么设计的吗还是我这里的`Bug`？
```json
{
  "name": "Icon多搜",
  "url": "[\"IconFont\",\"Iconfinder⁤\",\"FlatIcon\"]",
  "icon": "data:image/svg+xml;base64,PD94......"
}
```

**Comment by hoothin** [comment]
這是個bug，感謝反饋，已經修復了
